### ISSUE TITLE
GG Max - Bug - Creating a Standalone

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#3203]
I have just tried to create a standalone for my entry to the competition, end date tonight.
The latest version of Max is creating a error when i try to create a Standalone game, using the Steam version.
I have sent the map file to Rick as my entry for the competition instead. 
This is not my fault when Max creates errors in the last hour while i am trying to create the Standlone.

![image](https://user-images.githubusercontent.com/66874403/170798569-69b3b27a-708a-4d47-8c64-5fb5ca252642.png)

**Comment by synchromesh62** [comment]
I tried this with my game and can confirm the **exact same problem.**
Something is broken in todays build.

**Comment by Jisafu** [comment]
I have the same problem

**Comment by synchromesh62** [comment]
Yes seem to be getting quite a few reports of this now users are trying to test their standalones since the update.
It will probably be looked at in the morning now the weekend is out the way .

**Comment by ZakJudges** [comment]
Hi all, thanks for reporting this issue. I have added a fix and will post here when it is available for download

**Comment by ZakJudges** [comment]
Hi all, the fix should now be available for download. Thanks again

**Comment by synchromesh62** [comment]
@ZakJudges 
Experimental build updated earlier and it seems to be fine now.
Awesome ..Thanks

**Comment by Jisafu** [comment]
Hello,
I am an absolute beginner, where can i download it?

Thanks,
Jos Ichau

**Comment by Jisafu** [comment]
Hi,

It works perfect now.
I work with the Steam version.

Thanks,
Jos Ichau
